src/SignalR/clients/java/signalr/messagepack/build.gradle

```diff
@@ -0,0 +1,50 @@
+group 'com.microsoft.messagepack'
```

Comment:
This should probably be com.microsoft.messagepack.signalr or com.microsoft.signalr.messagepack

Reply:
https://github.com/dotnet/aspnetcore/pull/25253/commits/120bec6191a4e2702cec4cd8320f81b3b1e8e261

Reply:
Need to add a signalr folder with the messagepack folder under it :)

src\SignalR\clients\java\signalr\messagepack\src\main\java\com\microsoft\signalr\messagepack\

Folder structure is important for java.

Reply:
https://github.com/dotnet/aspnetcore/pull/25253/commits/18f368c896e6ab8691f1b99668bc0961280bd7b9